MagaluCloud/magalu
# Shell to fix links

## Description
Adds a shell and python script to run the `spec_check_links.py` for every openAPI yaml in `mgc/cli/openapis`

## Related Issues
Closes #713 


## How to test it
- Go to `mgc/cli/openapis` and modify the path of any parameter in a link to a incorrect path
- Try to commit it or any other change
- Expected: pre-commit should fail and apply a fix to the path
